.env

```diff
@@ -0,0 +1,9 @@
+OPEN_RUNTIMES_PROXY_ALGORITHM=round-robin
+OPEN_RUNTIMES_PROXY_EXECUTORS=mockoon1,mockoon2
+OPEN_RUNTIMES_PROXY_HEALTHCHECK=enabled
+OPEN_RUNTIMES_PROXY_HEALTHCHECK_INTERVAL=5000
+OPEN_RUNTIMES_PROXY_ENV=development
+OPEN_RUNTIMES_PROXY_SECRET=proxy-secret-key
+OPEN_RUNTIMES_PROXY_EXECUTOR_SECRET=executor-secret-key
```

Comment:
whats the different between these two ?

Reply:
`OPEN_RUNTIMES_PROXY_SECRET` is used between `appwrite` and `proxy`.
`OPEN_RUNTIMES_PROXY_EXECUTOR_SECRET` is used between `proxy` and `executor`.

2 different security layers. Appwrite shouldn't be able to talk to executor (unless specifically allowed), and executor should not know proxies secret (executor never talks to proxy, it just answers)